packages/service-proxy/README.md

```diff
@@ -0,0 +1,31 @@
+# @loopback/service-proxy
+
+This module provides a common set of interfaces for interacting with service
+oriented backends such as REST APIs, SOAP Web Services, and gRPC microservices.
+
+## Installation
+
+```
+$ npm install --save @loopback/service-proxy
+```
+
+## Usage
```

Comment:
This should be `Basic use`, see http://loopback.io/doc/en/contrib/README-guidelines.html

> This section is often titled “Usage,” but we prefer “Use” because it is a shorter and more common word, and means the same thing.